lang/ghc/Portfile

```diff
@@ -4,27 +4,28 @@ PortSystem          1.0
 PortGroup           gpg_verify 1.0
 
 name                ghc
-version             8.10.1
+version             8.10.2
 revision            0
 categories          lang haskell
 maintainers         {ieee.org:s.t.smith @essandess} openmaintainer
 license             BSD
 platforms           darwin
 supported_archs     x86_64
 
-description The Glorious Glasgow Haskell Compilation System
-long_description    \
-        The Glasgow Haskell Compiler is a robust,       \
-        fully-featured, optimising compiler and interactive \
-        environment for Haskell 98, GHC compiles Haskell to \
-        either native code or C.  It implements numerous    \
-        experimental language extensions to Haskell 98,     \
-        for example: concurrency, a foreign language interface, \
-        multi-parameter type classes, scoped type variables,    \
-        existential and universal quantification, unboxed   \
-        types, exceptions, weak pointers, and so on.        \
-        GHC comes with a generational garbage collector,    \
-        and a space and time profiler.
+description         The Glorious Glasgow Haskell Compilation System
+long_description    The Glasgow Haskell Compiler is a robust,\
+                    fully-featured, optimising compiler and\
+                    interactive environment for Haskell 98, GHC\
+                    compiles Haskell to either native code or C.  It\
+                    implements numerous experimental language\
+                    extensions to Haskell 98, for example:\
+                    concurrency, a foreign language interface,\
+                    multi-parameter type classes, scoped type\
+                    variables, existential and universal\
+                    quantification, unboxed types, exceptions, weak\
+                    pointers, and so on.  GHC comes with a\
+                    generational garbage collector, and a space and\
+                    time profiler.
```

Comment:
Please restore the formatting of the `long_description`. There is no need to make it thinner and waste more lines. If anything, you can make it thicker by indenting only 4 spaces.

Reply:
Well, I'd say it's a matter of style that's up to the maintainer. The way that Steve has changed it here is probably the closest to the MacPorts preferred style, except that Steve has omitted spaces before backslashes which we usually include, though it does not make a difference to the output.